test/hashing.test.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, it } from "bun:test";
+import { HASH_CONFIG } from "../src/utility/constants.ts";
+
+describe("Hashing Tests", () => {
+	it("should hash a password", () => {
+		const password = "1234567890";
+		const hashedPassword = Bun.password.hashSync(password, HASH_CONFIG);
+		expect(hashedPassword).toBeDefined();
+		expect(Bun.password.verify(password, hashedPassword)).toBeTruthy();
```

Comment:
`verify` returned een `Promise<boolean>`, omdat je deze nooit `await` zal deze `expect` *altijd* `truthy` zijn. Je moet `await` voor `Bun.password.verify` toevoegen.
```suggestion
		expect(await Bun.password.verify(password, hashedPassword)).toBeTruthy();
```